Lib/unittest/mock.py

```diff
@@ -402,6 +403,10 @@ def __init__(self, /, *args, **kwargs):
 class NonCallableMock(Base):
     """A non-callable version of `Mock`"""
 
+    # Store a mutex as a class attribute to protect concurrent accesses to
+    # a NonCallableMock's other attributes.
```

Comment:
Might be worth explaining briefly what we meant by "other attributes".

Reply:
I edited the comment!